api/tests/test_asset_log.py

```diff
@@ -43,6 +43,23 @@ def test_add_checkin(self):
             log_type="Checkin",
         )
         self.assertEqual(AssetLog.objects.count(), 3)
+        created_log = AssetLog.objects.filter(asset=self.test_other_asset).first()
+        self.assertEqual(created_log.log_type, "Checkin")
```

Comment:
could we use the constants defined [here](https://github.com/AndelaOSP/art-backend/blob/696da34d0de7ce234645e4df42899c9798352cf2/core/constants.py#L25) for "Checkin" and "Checkout"? we can also update the rest of the usage on this file.